thirdeye/thirdeye-frontend/app/pods/manage/alert/tune/controller.js

```diff
@@ -262,7 +277,7 @@ export default Controller.extend({
       }
 
       // Number the list
-      if (this.get('isTunePreviewActive')) {
+      if (filteredAnomalies) {
```

Comment:
Is this check needed? Can this ever be null or undefined? (based on the call above, it looks like this will always be an array)

Reply:
True - not needed.